test/testScenarios.ts

```diff
@@ -100,9 +99,6 @@ describe("All good scenario", async function () {
       await balotFromSafe.balanceOf(ADDRESSES.safe)
     );
     expect(newSafeBalance).to.equal(initialSafeBalance + 300);
-    expect((await balotFromSafe.owner()).toUpperCase()).to.equal(
-      ADDRESSES.safe.toUpperCase()
-    );
 
     console.debug(
       `SafeMintRange gas used: ${(await safeMintRangeTx.wait()).gasUsed}`
```

Comment:
To make a complete scenario, I'd add a 4th step exactly as in the failing case, to send ownership back to the safe.

Reply:
ok